src/api/v2/db/ldap/directory-dao.js

```diff
@@ -53,7 +53,7 @@ const mapQuery = (endpointQuery) => {
       case 'alternatePhoneNumber':
       case 'faxNumber':
       case 'officeAddress': {
-        return `(${ldapKey}=*${value}*)`;
+        return `${ldapKey}=*${value}*`;
```

Comment:
@kalcicd But looks like other cases also contain the outside parentheses. Weren't they broke the query?

Reply:
The default case does not have any parenthesis, the only difference being that the default case does not have asterisks surrounding the value. The cases of `phoneNumber` and `fuzzyName` are surrounded with parenthesis because they are compound queries and need the parenthesis to seperate them. For example, `phoneNumber=123` sends the ldap query `(&(|(telephoneNumber=*123*)(osuAltPhoneNumber=*123*)(facsimileTelephoneNumber=*123*)))`

Reply:
The cases that were broken earlier were producing ldap queries like`(&((telephoneNumber=*404*)))`. There's an unnecessary set of parentheses in there, which ldap doesn't like, resulting in the error